src/extension.ts

```diff
@@ -69,8 +69,10 @@ function activateNoHieCheck(context: ExtensionContext) {
       .replace('${workspaceFolder}', workspaceFolder.uri.path)
       .replace('${workspaceRoot}', workspaceFolder.uri.path)
       .replace('${HOME}', os.homedir)
-      .replace('${home}', os.homedir)
-      .replace('~', os.homedir);
+      .replace('${home}', os.homedir);
+    if (customWrapperPath.startsWith('~')) {
+      customWrapperPath = customWrapperPath.replace('~', os.homedir);
```

Comment:
So if it starts with a `~`, let’s replace all tildes? =P

This is bike-shedding, really (on my part!), but maybe:

```
> "~/.ab~ac".replace(/^~/, "»HOME«")

"»HOME«/.ab~ac"
```

JavaScript has regex literals… (or at least my browser’s console has them — above).

Reply:
Nice! Then I can do away with the 'startsWith` check. I'll add it in the next commit.